modules/table/src/integrationTest/java/org/apache/ignite/internal/raftsnapshot/ItTableRaftSnapshotsTest.java

```diff
@@ -291,9 +178,11 @@ private void testLeaderFeedsFollowerWithSnapshot(String storageEngine) throws Ex
 
         transferLeadershipOnSolePartitionTo(2);
 
-        List<IgniteBiTuple<Integer, String>> rows = queryWithRetry(2, "select * from test", ItTableRaftSnapshotsTest::readRows);
+        assertThat(getVia(2, 1), is("one"));
+    }
 
-        assertThat(rows, is(List.of(new IgniteBiTuple<>(1, "one"))));
+    private @Nullable String getVia(int clusterNode, int key) {
```

Comment:
This method's name is confusing to me, can we rename it to `getFromNode`, for example?